### DISCUSSION TITLE
provide way to NOT set release to sdkman as default

### DISCUSSION THREAD

**Discussion by maxandersen** [jreleaser/jreleaser#1304] [Q&A]
just got aware we by accident been publishing quarkus 3 alpha and betas into sdkman as default versions.

For now we work around it using:

```
curl -X PUT -H "Consumer-Key: CONSUMER_KEY" -H "Consumer-Token: CONSUMER_TOKEN" -H "Content-Type: application/json" -H "Accept: application/json" -d '{"candidate": "quarkus", "version": "2.16.5.Final"}' https://vendors.sdkman.io/default
```

how can we make jreleaser stop setting a release as default?

**Comment by aalmiray** [comment]
This is already possible by setting the value of `sdkman.command` to `MINOR`. See https://github.com/jreleaser/jreleaser.github.io/commit/0d83386401a45f709208cda341781bbd62bbf3b9
